app/javascript/dashboard/routes/dashboard/conversation/search/SwitchLayout.vue

```diff
@@ -4,18 +4,11 @@
     class="layout-switch__container"
     @click="toggle"
   >
-    <div
-      class="layout-switch__btn left"
-      :class="{ active: !isOnExpandedLayout }"
-    >
-      <fluent-icon icon="panel-sidebar" :size="16" />
-    </div>
-    <div
-      class="layout-switch__btn right"
-      :class="{ active: isOnExpandedLayout }"
-    >
-      <fluent-icon icon="panel-contract" :size="16" />
-    </div>
+    <fluent-icon
```

Comment:
>  Can we use `woot-button` here? Might need to change CSS for rotating the icon
> 

@nithindavid thought of using this, but using the rotate on the button itself would rotate the entire button instead of that icon

Reply:
@fayazara We could use this
```

  &.expanded .icon {
    transform: rotate(180deg);
  }
  
  ```

Reply:
@nithindavid Updated, please have a look